### PR TITLE
Examples - polish & linting

### PR DESCRIPTION
**Pull request by ttodua**
continual of https://github.com/ccxt/ccxt/pull/14798/files

### CODE REVIEW

File: examples/ts/fetch-orderbook.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
-import { binance } from '../../ts/ccxt.js'
```
Comment (carlosmiei) [inline examples/ts/fetch-orderbook.ts:1]: @ttodua I was doing this specific import on purpose to showcase that we don't need to import the entire package to use only one or two exchanges

Comment (ttodua) [reply examples/ts/fetch-orderbook.ts:1]: ah ok.
which way do you suggest, so it remained transpilable.

File: examples/ts/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
```
Comment (carlosmiei) [inline examples/ts/.eslintrc:1]: why do we need a custom `eslintrc` here?

Comment (ttodua) [reply examples/ts/.eslintrc:1]: it enables linting in examples folder too for TS. is not that good?

Comment (ttodua) [reply examples/ts/.eslintrc:1]: i've copied it from `ts` and made only few edits

File: examples/ts/fetch-orderbook.ts (hunk 1 above)
Comment (ttodua) [reply examples/ts/fetch-orderbook.ts:1]: yes, we can have both examples inside same file, maybe we shouldnt create separate files just to teach users how to import. we can smth like this in same file:

```
import { binance } from '../../ts/ccxt.js'; // preferred approach
import ccxt from '../../ts/ccxt.js';
...
b1 = new binance(); // preferred approach
b2 = new ccxt['binance'];
```
however, I think we will need some changes in transpiler (or maybe in AST-Transpiler)?

Comment (carlosmiei) [reply examples/ts/fetch-orderbook.ts:1]: @ttodua Here the goal is only to show the import so the name can express that `import-one-exchange-esm-example` for instance.

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**